venues/OpenReview.net/Support/deployProcess.py

```diff
@@ -5,7 +5,7 @@ def process(client, note, invitation):
     conference = openreview.helpers.get_conference(client, note.forum)
     print(conference.get_id())
     forum = client.get_note(id=note.forum)
-    comment_readers = forum.content['Contact Emails']
+    comment_readers = forum.content['Contact Emails'][:]
```

Comment:
Why the change? Don't they do the same thing?

Reply:
Earlier, "comment_readers" was a reference to the list of contact emails. Any changes to "comment_readers" would actually be changes to the forum.content['Contact Emails']. Also, I am doing a client.post_note(forum) some statements below this. As expected, this was making changes to the "Contact Emails" in the forum's content which was never the intention.
Using [:] creates a shallow copy of the list and assigns this copy to comment_readers, which ensures that changes to comment_readers would not affect the forum object.